api/v1beta1/azureclusteridentity_webhook.go

```diff
@@ -26,3 +32,30 @@ func (c *AzureClusterIdentity) SetupWebhookWithManager(mgr ctrl.Manager) error {
 		For(c).
 		Complete()
 }
+
+// +kubebuilder:webhook:verbs=create;update,path=/validate-infrastructure-cluster-x-k8s-io-v1beta1-azureclusteridentity,mutating=false,failurePolicy=fail,matchPolicy=Equivalent,groups=infrastructure.cluster.x-k8s.io,resources=azureclusteridentities,versions=v1beta1,name=validation.azureclusteridentity.infrastructure.cluster.x-k8s.io,sideEffects=None,admissionReviewVersions=v1;v1beta1
+
```

Comment:
extra newline?

Reply:
I saw that other webhook files had the kubebuilder command separated in its own block like [this](https://github.com/willie-yao/cluster-api-provider-azure/blob/7db161723f1aaadb4d79f211c35e788139ece18f/api/v1beta1/azureclustertemplate_webhook.go#L40). Would this hold true for this command as well?

Reply:
true, pls ignore my comment